pennylane_lightning/lightning_qubit.py

```diff
@@ -286,11 +309,7 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
         state_vector = StateVectorC64(ket) if use_csingle else StateVectorC128(ket)
 
         jac = adj.adjoint_jacobian(
-            state_vector,
-            obs_serialized,
-            ops_serialized,
-            tp_shift,
-            tape.num_params,
+            state_vector, obs_serialized, ops_serialized, tp_shift, tape.num_params
```

Comment:
As above, this is also by black. As it seems like a way to prevent black from reformatting this (https://stackoverflow.com/questions/58584413/black-formatter-ignore-specific-multi-line-code), I will try it.